central/debug/service/db_diagnostics.go

```diff
@@ -0,0 +1,82 @@
+package service
+
+import (
+	"context"
+	"os/exec"
+	"strings"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/stackrox/rox/central/globaldb"
+)
+
+const (
+	unableToGetConnectionInfo = "Unable to get the connection string"
+)
+
+func buildDBDiagnosticData(ctx context.Context, dbConfig *pgxpool.Config, dbPool *pgxpool.Pool) centralDBDiagnosticData {
+	diagnosticData := centralDBDiagnosticData{}
+
+	// Add the database version if Postgres
+	diagnosticData.Database = "PostgresDB"
```

Comment:
Define it in a enum or const. This could be used in other place for database type.